services/worker-manager/test/provisioner_test.js

```diff
@@ -86,6 +86,7 @@ helper.secrets.mockSuite(testing.suiteName(), ['azure'], function(mock, skipping
       workers: [
         {
           workerPoolId: 'ff/ee',
+          existingCapacity: 0,
```

Comment:
I don't think this belongs here, as this is a row to be added to the Workers table, and that has no "existingCapacity" column.  Have I misunderstood?